src/ast/ddl.rs

```diff
@@ -678,6 +678,25 @@ impl fmt::Display for ColumnOption {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit, VisitMut))]
+pub struct StructField {
```

Comment:
Can you please add docstrings that explain what this struct is for (with an example SQL snippet)